client/synapseJavaClient/src/main/java/org/sagebionetworks/client/SynapseClient.java

```diff
@@ -1593,7 +1554,27 @@ MembershipInvtnSubmission createMembershipInvitation(
 	 * @throws SynapseException
 	 */
 	Count getOpenMembershipInvitationCount() throws SynapseException;
-	
+
+	/**
+	 * Verify whether the inviteeEmail of the indicated MembershipInvitation is associated with the authenticated user.
+	 * If it is, the response body will contain an InviteeVerificationSignedToken.
+	 * If it is not, the response body will be null.
+	 * @param membershipInvitationId

```

Comment:
Note:  This will update if the controller is changed.

Reply:
Fixed

Reply:
"getInviteeVerificationSignedToken()"